### PR TITLE
Xdebug configuration

### PR DESCRIPTION
**Pull request by shucon**
Enables Xdebug debugging on host

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -646,3 +651,7 @@ services:
 networks:
   williamdeslocal:
     driver: bridge
+    ipam:
```
Comment (williamdes) [inline docker-compose.yml:691]: where is this from ?

File: docker-compose.yml [hunk 2]
```diff
@@ -223,6 +223,11 @@ services:
     restart: on-failure:2
     mem_limit: 1g
     working_dir: /mnt/Dev
+    environment:
+      PHP_EXTENSION_XDEBUG: 1
+      PHP_INI_XDEBUG__REMOTE_PORT: 10000
+      PHP_IDE_CONFIG: "serverName=plumbus.com"
```
Comment (williamdes) [inline docker-compose.yml]:
```suggestion
      PHP_IDE_CONFIG: "serverName=plumbus.com"
```
Why plumbus.com ?

Comment (shucon) [reply docker-compose.yml]: I think this line is not required at all

File: docker-compose.yml (hunk 1 above)
Comment (shucon) [reply docker-compose.yml:691]: I took it from stackoverflow to make the gateway same every time

File: docker-compose.yml [hunk 3]
```diff
@@ -223,6 +223,10 @@ services:
     restart: on-failure:2
     mem_limit: 1g
     working_dir: /mnt/Dev
+    environment:
+      PHP_EXTENSION_XDEBUG: 1
```
Comment (williamdes) [inline docker-compose.yml]: I will add an ENV for that after merging

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
I will update the templates because this file is a generated one, see: https://github.com/wdesportes/phpmyadmin-local-setup/tree/master/compose-templates

**Comment by shucon** [comment]
> I will update the templates because this file is a generated one, see: https://github.com/wdesportes/phpmyadmin-local-setup/tree/master/compose-templates

Okay, perfect

File: docker-compose.yml [hunk 4]
```diff
@@ -646,3 +650,7 @@ services:
 networks:
   williamdeslocal:
     driver: bridge
+    ipam:
+      config:
+        - subnet: 172.19.0.0/16
+        - gateway: 172.19.0.1
```
Comment (williamdes) [inline docker-compose.yml]: https://github.com/docker/compose/issues/4456#issuecomment-278779861

**Review by williamdes — APPROVED** [review]
:100:

**Pull request merged**